src/components/molecules/Visualizer/Engine/Cesium/Box/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { Cartesian3, Color, Plane as CesiumPlane, TranslationRotationScale } from "cesium";
+import React, { useMemo, useEffect, memo, useState } from "react";
+
+import { LatLng, toColor } from "@reearth/util/value";
+
+import type { Props as PrimitiveProps } from "../../../Primitive";
+
+import { Side } from "./Side";
+
+export type Props = PrimitiveProps<Property>;
+
+export type BoxStyle<C = Color | string> = {
+  fillColor?: C;
+  outlineColor?: C;
+  outlineWidth?: number;
+  fill?: boolean;
+  outline?: boolean;
+};
+
+export type Property = {
+  default?: {
+    position?: LatLng;
+    location?: LatLng;
+    height?: number;
+    dimensions?: {
+      x: number;
+      y: number;
+      z: number;
+    };
+  } & BoxStyle<string>;
+};
```

Comment:
```suggestion
export type Property = {
  default?: {
    location?: LatLngHeight;
    width?: number;
    height?: number;
    length?: number;
    heading?: number;
    pitch?: number;
    roll?: number;
    fillColor?: string;
    outlineColor?: string;
    outlineWidth?: number;
    fill?: boolean;
    outline?: boolean;
  };
};
```

Reply:
This was fixed in [refactor: fix property type](https://github.com/reearth/reearth-web/pull/338/commits/9cecd3a70245748ab056c2d9ce0a54dda57be7b0)